drivers/hmis/app/graphql/types/forms/pick_list_option.rb

```diff
@@ -234,7 +234,8 @@ def self.coc_picklist(selected_project)
     end
 
     def self.geocodes_picklist
-      state = ENV['RELEVANT_COC_STATE']
+      # NOTE: HMIS currently only supports one state installations
+      state = ENV['RELEVANT_COC_STATE']&.split(',')&.first
```

Comment:
Maybe it's too late, but `ENV['RELEVANT_COC_STATE']&.split(',')` could be a constant somewhere.